eth/catalyst/api.go

```diff
@@ -218,14 +218,14 @@ func (api *ConsensusAPI) verifyPayloadAttributes(attr *engine.PayloadAttributes)
 
 	// Verify withdrawals attribute for Shanghai.
 	withdrawalsCheck := func(b *big.Int, t uint64) bool {
-		return c.IsEnabledByTime(c.GetEIP4895TransitionTime, &t)
+		return c.IsEnabledByTime(c.GetEIP4895TransitionTime, &t) || c.IsEnabled(c.GetEIP4895Transition, b)
```

Comment:
this wasn't in place from the last merge and I added it now @meowsbits. Any reason not to do it?

Reply:
I don't think there's any reason not -- thanks for the catch.